sdk/communication/azure-communication-sms/README.md

```diff
@@ -128,15 +132,34 @@ for (SmsSendResult result : sendResults) {
 
 ## Troubleshooting
 
-All SMS service operations will throw an exception on failure.
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L104-L112 -->
+- SMS operations will throw an exception if the request to the server fails.
+- Exceptions will not be thrown if the error is caused by an individual message, only if something fails with the overall request.
+- Please use the `Successful` flag to validate each individual result to verify if the message was sent.
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L248-L273 -->
 ```java
+SmsClient smsClient = createSmsClientUsingAzureKeyCredential();
+
+SmsSendOptions options = new SmsSendOptions();
+options.setDeliveryReportEnabled(true);
+options.setTag("Tag");
```

Comment:
Can we add line 142 -144 into the try-catch since it is part of creating the sendWithResponse call?

Reply:
actually while we're here, if you can set the tag according to the .NET example which sets it to "marketing", that would be good. more consistency across samples!

Reply:
I think Tag makes more like "here is where you put the tag"

Reply:
for the try and catch is done

Reply:
> I think Tag makes more like "here is where you put the tag"

hmm, I think that the method name already does a good job of explaining that, doesn't it?
I agreed with Minnie that having an example of a possible tag value was useful.

Reply:
changed